include/deal.II/lac/petsc_precondition.h

```diff
@@ -146,6 +146,14 @@ namespace PETScWrappers
                         const AdditionalData &additional_data = AdditionalData());
 
     /**
+     * Same as above but without setting a matrix to form the preconditioner.
+     * Intended to be used with SLEPc objects.
+     */
+    PreconditionJacobi (const MPI_Comm communicator,
+                        const AdditionalData &additional_data = AdditionalData());
```

Comment:
Is there any possibility to catch the uninitialized case by querying PETSc information (without introducing additional state variables; to maintain such additional data structures is not worth the pain)? If you do not find anything, I think this is fine to merge anyway.